Code/Framework/AzCore/AzCore/EBus/Event.inl

```diff
@@ -207,6 +207,32 @@ namespace AZ
     }
 
 
+    template <typename... Params>
+    auto Event<Params...>::StealHandlers(Event&& other) -> Event&
+    {
+        decltype(other.m_handlers) handlers = AZStd::move(other.m_handlers);
+        decltype(other.m_addList) addList = AZStd::move(other.m_addList);
```

Comment:
Can't `auto` be used here instead of the `decltype`?

Reply:
Yes it can be used here.